### DISCUSSION TITLE
Welcome to CATs-Eurosynth Discussions!

### DISCUSSION THREAD

**Discussion by mzuelch** [mzuelch/CATs-Eurosynth#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by modulove** [comment]
Hi there, 

we are Modulove, a collective of DIY enthusiasts. We (sasha, kevin and me) started our lab because of the corona crisis and lockdowns.

We like the open development and interaction of eurorack modular and we're happy to see Panel / PCB versions of Hagiwos modules being released into the wild. It an amazing process building, designing and prototyping panels and modules. That kept us motivated.
We are currently working on some firmware updates to bring more functionality :) 

We are really greatful and share hagiwos intention to make diy Eurorack modules more accessible and affordable for those who may otherwise avoid this area of eurorack. 

Happy patching! 
Linus
